requirements_dev.txt

```diff
@@ -0,0 +1,4 @@
+mypy
```

Comment:
What is this file `requirements_dev.txt` being used for?

Reply:
`requirements_dev.txt` are the pip-requirements for development.
I seperated them because these are not reqired for production. 
I can merge them if you'd like..

Reply:
Aha, no this is fine. Good idea!